scripts/player.gd

```diff
@@ -0,0 +1,39 @@
+extends KinematicBody2D
+var vect = Vector2(0,0)
+var up = Vector2(0,-1)
```

Comment:
can we give these variables types so that others can know what this variable type is

var up:Vector = Vector2(0,-1)